app/assets/javascripts/surveys.js

```diff
@@ -345,6 +345,25 @@
           ]
         },
         allowedOnMobile: true
+      },
+      {
+        identifier: 'Business_iteration 1',
+        surveyType: 'url',
+        frequency: 6,
+        startTime: new Date('June 29, 2018').getTime(),
+        endTime: new Date('July 4, 2018').getTime(),
+        url: 'https://GDSUserResearch.optimalworkshop.com/treejack/wu86k8e2?c={{currentPath}}',
+        templateArgs: {
+          title: 'Help us make things easier to find on GOV.UK',
+          surveyCta: 'Answer 2 quick questions',
+          surveyCtaPostscript: 'This activity will open in a separate window'
+        },
+        activeWhen: {
```

Comment:
is this double activeWhen correct?

Reply:
Nope, good spot.

I've corrected that now.